test/c_api_tests.cpp

```diff
@@ -94,6 +94,43 @@ TEST(CAPITests, TokenizerCAPI) {
 #endif
 }
 
+TEST(CAPITests, ChatTemplate) {
+#if TEST_PHI2
+  auto config = OgaConfig::Create(PHI2_PATH);  // just to create tokenizer (phi-2 does not have a chat template)
+  auto model = OgaModel::Create(*config);
```

Comment:
```suggestion
  auto tokenizer = OgaTokenizer::Create(*OgaModel::Create(PHI2_PATH));
```
See if this works, just to make it even simpler.